packaging_automation/citus_package.py

```diff
@@ -13,11 +13,16 @@
 from parameters_validation import non_blank, non_empty, validate_parameters
 
 from .common_tool_methods import (run_with_output, PackageType, transform_key_into_base64_str,
-                                  get_gpg_fingerprints_by_name, str_array_to_str)
+                                  get_gpg_fingerprints_by_name, str_array_to_str,
+                                  get_supported_postgres_release_versions, get_supported_postgres_nightly_versions,
+                                  DEFAULT_ENCODING_FOR_FILE_HANDLING, DEFAULT_UNICODE_ERROR_HANDLER)
 from .packaging_warning_handler import validate_output
 
 GPG_KEY_NAME = "packaging@citusdata.com"
 
+POSTGRES_VERSION_FILE = "supported-postgres.txt"
```

Comment:
See my comments on this file name at https://github.com/citusdata/packaging/pull/738#discussion_r719351516

Reply:
Will be fixed after completion of https://github.com/citusdata/packaging/pull/738 review

Reply:
bump

Reply:
now creating packaging-test images. After that I will fix it